project/SporesBuild.scala

```diff
@@ -9,8 +9,8 @@ object SporesBuild extends Build {
     organization := "org.scala-lang.modules",
     organizationName := "LAMP/EPFL",
     organizationHomepage := Some(new URL("http://lamp.epfl.ch")),
-    version      := "0.2.2-SNAPSHOT",
-    scalaVersion := "2.11.6"
+    version      := "0.2.4-M1",
```

Comment:
This was a quick workaround to avoid overriding `0.2.3`, which i had already published and used locally. Changed.